Document/0x04g-Testing-Cryptography.md

```diff
@@ -89,7 +81,7 @@ Other uses of cryptography require careful adherence to best practices:
 
 #### Remediation
 
-Periodically ensure that the cryptography has not become obsolete. Some older algorithms, once thought to require years of computing time, can now be broken in days or hours. This includes MD4, MD5, SHA1, DES, and other algorithms that were once considered as strong. Examples of currently recommended algorithms<sup>[1], [2]</sup>:
+Periodically ensure that the cryptography has not become obsolete. Some older algorithms, once thought to require years of computing time, can now be broken in days or hours. This includes MD4, MD5, SHA1, DES, and other algorithms that were once considered as strong. Examples of currently recommended algorithms ([NSA](https://cryptome.org/2016/01/CNSA-Suite-and-Quantum-Computing-FAQ.pdf "Commercial National Security Algorithm Suite and Quantum Computing FAQ"), [NIST](http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-57pt1r4.pdf "NIST Special Publication 800-57")), :
```

Comment:
> "NIST Special Publication 800-57"

I think the title "Recommendation for Key Management" is more descriptive.

> )), :

I think the comma should be removed here.